scipy/special/sf_error.cc

```diff
@@ -8,3 +8,24 @@
 extern "C" {
 #include "sf_error.c"
 }
+
+
+namespace scipy {
+    namespace special {
+	void set_error(const char *func_name, int code, const char *fmt, ...) {
```

Comment:
Best practice for this is to just do `void scipy::special::set_error(...) { ... }` without the namespaces. That should work.

Reply:
It doesn't work if I just do that directly.

```
../scipy/special/sf_error.cc:13:6: error: ‘scipy’ has not been declared
   13 | void scipy::special::set_error(const char *func_name, int code, const char *fmt, ...) {
      |      ^~~~~
```

Reply:
I think you just need to do `#include "error.h"` in `sf_error.cc`, then all will be well.